test/uk/gov/hmrc/teamsandrepositories/GitRepositorySpec.scala

```diff
@@ -50,43 +38,51 @@ class GitRepositorySpec extends WordSpec with Matchers with OptionValues {
   def githubLink(url: String) = Link("github-com", "GitHub.com", url)
 
   "repoGroupToRepositoryDetails" should {
-    "create links for open libraries" in {
+    "create links for libraries" in {
       val repo =
         GitRepository(
           "a-library",
           "Some Description",
           "https://github.com/org/a-library",
           now,
           now,
-          repoType = RepoType.Library,
-          language = Some("Scala")
+          isPrivate = true,
+          repoType  = RepoType.Library,
+          language  = Some("Scala")
         )
 
-      val repoDetails = RepositoryDetails.create(repo, Seq("teamName"), urlTemplates)
+      val repoDetails = RepositoryDetails.create(repo, Seq("a-team"), urlTemplates)
 
       repoDetails.githubUrl shouldBe githubLink("https://github.com/org/a-library")
-      repoDetails.ci        shouldBe List(Link("open1", "open 1", "http://open/a-library"))
+      repoDetails.ci        shouldBe List(Link("Build", "Build", "https://build.tax.service.gov.uk/job/a-team/job/a-library"))
     }
 
-    "create links for open services" in {
+    "create ci links for libraries belonging to multiple teams" in {
       val repo =
         GitRepository(
-          "a-frontend",
+          "a-library",
           "Some Description",
-          "https://github.com/org/a-frontend",
+          "https://github.com/org/a-library",
           now,
           now,
-          repoType = RepoType.Service,
-          language = Some("Scala")
+          isPrivate = true,
+          repoType  = RepoType.Library,
+          language  = Some("Scala")
         )
 
-      val repoDetails = RepositoryDetails.create(repo, Seq("teamName"), urlTemplates)
+      val repoDetails = RepositoryDetails.create(repo, Seq("a-team", "another-team"), urlTemplates)
 
-      repoDetails.githubUrl shouldBe githubLink("https://github.com/org/a-frontend")
-      repoDetails.ci        shouldBe List(Link("open1", "open 1", "http://open/a-frontend"))
+      repoDetails.githubUrl shouldBe githubLink("https://github.com/org/a-library")
+      repoDetails.ci shouldBe List(
+        Link("a-team Build", "a-team Build", "https://build.tax.service.gov.uk/job/a-team/job/a-library"),
+        Link(
+          "another-team Build",
+          "another-team Build",
+          "https://build.tax.service.gov.uk/job/another-team/job/a-library")
+      )
     }
 
-    "create links for private libraries" in {
+    "not create ci links for libraries belonging to no teams" in {
```

Comment:
The sentence describing the scenario is a little confusing due to the "not" at the beginning.  Could you reword it to something that reads a little easier such as: "libraries without a team should not generate any links"

Reply:
It's intended to be read as
```
"repoGroupToRepositoryDetails" should {
"not create ci links for libraries belonging to no teams" in {
```